Helm function "divf" not defined

<!-- Thanks for filing an issue! Before hitting the button, please answer these questions. It's helpful to search the existing GitHub issues first. It's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of-->

**Is this a request for help?**: No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

<!--
If this is a BUG REPORT, please:
  - Fill in as much of the template below as you can.  If you leave out
    information, we can't help you as well.

If this is a FEATURE REQUEST, please:
  - Describe *in detail* the feature/behavior/change you'd like to see.

In both cases, be ready for followup questions, and please respond in a timely
manner.  If we can't reproduce a bug or think a feature already exists, we
might close your issue.  If we're wrong, PLEASE feel free to reopen it and
explain why.
-->

**Version of Helm and Kubernetes**:
helm: `v3.5.4`
kubernetes: `v1.21.1`

<!-- To get current helm, kubernetes and ct versions:
helm version
kubectl version
ct version
and paste output here.
-->

**What happened**:
On `ct lint`, the validation fails with `[ERROR] templates/: parse error at (chart-name/templates/some-manifest.yaml:20): function "divf" not defined`.


**What you expected to happen**:
I believe it should work with the latest helm version, at least it does locally.

**How to reproduce it** (as minimally and precisely as possible):
have a template with the `divf` function [from sprig](http://masterminds.github.io/sprig/mathf.html). The current chart-testing container image will fail while validating.

**Anything else we need to know**:
I'll submit a PR to bump the dependencies

--- a/Dockerfile
+++ b/Dockerfile
@@ -17,19 +17,19 @@ LABEL yamllint_version=$yamllint_version
 RUN pip install "yamllint==$yamllint_version"
 
 # Install Yamale YAML schema validator
-ARG yamale_version=3.0.6
+ARG yamale_version=3.0.7
 LABEL yamale_version=$yamale_version
 RUN pip install "yamale==$yamale_version"
 
 # Install kubectl
-ARG kubectl_version=v1.21.1
+ARG kubectl_version=v1.21.3
 LABEL kubectl_version=$kubectl_version
 RUN curl -LO "https://storage.googleapis.com/kubernetes-release/release/$kubectl_version/bin/linux/amd64/kubectl" && \
     chmod +x kubectl && \
     mv kubectl /usr/local/bin/
 
 # Install Helm
-ARG helm_version=v3.5.4
+ARG helm_version=v3.6.3
 LABEL helm_version=$helm_version
 RUN curl -LO "https://get.helm.sh/helm-$helm_version-linux-amd64.tar.gz" && \
     mkdir -p "/usr/local/helm-$helm_version" && \

